submissions/vakulinina/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;600;700&display=swap" rel="stylesheet">
+  <title>Book your seat</title>
+</head>
+
+<body>
+  <main>
+    <header>
+      <h1>Buy tickets</h1>
```

Comment:
`h1` better use for the movie title.

Reply:
Which tag should I use for the "Buy tickets" in the `header` then?